### ISSUE TITLE
Device names in Homebridge not updating to reflect Unifi changes

### ISSUE THREAD

**Issue by nickhandel** [hjdhjd/homebridge-unifi-protect#967]
This plugin is a work of art.  I'm in awe!  Thank you so much.

**Describe The Problem:**

I changed the name of a device in the Unifi app (for example, I moved a camera from one place to another and renamed it to better reflect its view).  This name change doesn't seem to be reflected in the Homebridge UI.  I use the 'accessories' tab to organise things at the Homebridge layer.  I do appreciate that this may be intended, but it's a useful step for me to allow me to keep track of things across multiple plugins - some are less well behaved than yours when it comes to the HomeKit app.

Is there a way to "refresh" the names of devices as specified in the Unifi console?

**To Reproduce:**

Adopt a new camera in a Unifi setup, with a given name - verify that the name is correct in the HomeBridge UI.
Change the name in Unifi.  The name change is not reflected in Homebridge.

**Logs:**

```
Show the Homebridge logs here.
Remove any sensitive information.
```

**Homebridge Configuration:**

```json
          {  "name": "UniFi Protect",
            "_bridge": {
                "username": "0E:C4:66:AC:1A:47",
                "port": 54703
            },
            "platform": "UniFi Protect"
        }
```

**Screenshots:**

**Environment:**

* **Homebridge Version:** v1.6.1
* **Node Version:** v18.16.1
* **Homebridge-UniFi-Protect Plugin Version:** 6.11.1
* **UniFi Protect Version:**  Protect 2.8.35
* **UniFi Protect Controller Type:**  UNVR
* **UniFi Protect Controller Firmware Version:**   UniFi OS 3.1.14
* **Apple Device and iOS / macOS / iPadOS / tvOS Version:** various all latest
* **Operating System and OS Version:**  macOS Ventura

**Comment by hjdhjd** [comment]
Thanks so much for the compliment - means a lot! 😄 😊 

The behavior is definitely intentional...but also, until relatively recently (last one or two major iOS releases) it's been a technical impossibility - HomeKit didn't allow accessories to rename themselves in HomeKit after providing an initial default name.

It's on my list to experiment with at some point and see if we can truly do this in a way that provides the experience I would want, where device names would just "stay in sync". There are a bunch of behind-the-scenes challenges in doing so, but it's a topic of interest of mine, albeit not high on the list right now.

TL;DR: yep...current behavior is intentional. Yep...would like to enable the ability to do this as a feature option, but definitely not the default behavior. No ETA or commitment...but I'm working through some other priorities right now. Once I'm comfortable that I've got something robust that can work well for folks, I'll bring it forward.

There is a yucky workaround (that I actually use when I need to do this):

* disable the camera in the HBUP webUI and restart HBUP.
* rename the camera in the Protect webUI/app if you haven't do so.
* enable the camera in the HBUP webUI and restart HBUP.

Camera will be removed and re-added to HomeKit using the new name. Obviously a very suboptimal user experience, but gets the job done.

Thanks again for the feedback and the feature suggestion.

**Comment by nickhandel** [comment]
Thanks so much for the immediate response!  Makes perfect sense; once my installation has settled and I've actually moved in I shall follow those steps as suggested to re-sync everything.

For what it's worth, I think that giving up your time for the benefit of others like this reflects humanity at its best!  Thanks again.

**Comment by hjdhjd** [comment]
Thanks a bunch and stay tuned for more goodness in the future. 😄 

Appreciate your kind words - spread the word to others about the awesomeness of HBUP, if you enjoy it!
